src/aks-preview/azext_aks_preview/agentpool_decorator.py

```diff
@@ -475,6 +475,66 @@ def get_skip_gpu_driver_install(self) -> bool:
 
         return skip_gpu_driver_install
 
+    def get_enable_secure_boot(self) -> bool:
```

Comment:
Please also add `# pylint: disable=too-many-public-methods` to line 55 to fix the failed style CI
```
ERROR: ************* Module azext_aks_preview.agentpool_decorator
src/aks-preview/azext_aks_preview/agentpool_decorator.py:56:0: R0904: Too many public methods (23/20) (too-many-public-methods)
```



https://dev.azure.com/azclitools/public/_build/results?buildId=135957&view=logs&j=506b8f13-8269-5210-246e-a3964e53141e&t=db7247cb-300d-5b59-fd6c-6258fad20b89&l=132